statemachine_test.go

```diff
@@ -233,6 +236,65 @@ func TestStateMachine_DiscardGarbage(t *testing.T) {
 	)
 }
 
+type stateMachineTest struct {
+	run []string
+}
+
+func (sm *stateMachineTest) Check(t *T) {}
+
+func (sm *stateMachineTest) ActionT(t *T) {
+	sm.run = append(sm.run, "ActionT")
+}
+
+func (sm *stateMachineTest) ActionTB(t TB) {
+	if len(sm.run) > 2 {
+		// Add a value to run that isn't expected to ensure the post-action check is skipped.
+		sm.run = append(sm.run, "Post-Skip")
+		t.Skip()
+	}
+
+	sm.run = append(sm.run, "ActionTB")
+}
+
+func TestStateMachineActions(t *testing.T) {
+	t.Run("Check", MakeCheck(func(t *T) {
+		sm := &stateMachineTest{}
+		actions := StateMachineActions(sm)
+
+		actionT, ok := actions["ActionT"]
+		if !ok {
+			t.Fatalf("ActionA missing")
+		}
+		actionTB, ok := actions["ActionTB"]
+		if !ok {
+			t.Fatalf("ActionTB missing")
+		}
+
+		var want []string
+		for i := 0; i < Int().Draw(t, "ActionT acount"); i++ {
```

Comment:
`acount` -- typo, here and in the loop below.

Reply:
Fixed typo, thanks